src/custom/state/gas/atoms.ts

```diff
@@ -1,6 +0,0 @@
-import { atom } from 'jotai'
-import { DEFAULT_GP_PRICE_STRATEGY } from 'constants/index'
-
-export type GpPriceStrategy = 'COWSWAP' | 'LEGACY'
-
-export const gasPriceStrategyAtom = atom<GpPriceStrategy>(DEFAULT_GP_PRICE_STRATEGY)
```

Comment:
If you are removing this, you should update the internal docs https://www.notion.so/cownation/00f45ee4a4614f62bdd914998f49a894?v=518cff14786b4e8190a177a23c988571&pvs=4

![image](https://user-images.githubusercontent.com/43217/223696403-1b44acf0-d61e-4ccf-9d59-77d4f842caa8.png)

cc @anxolin for more context and I was never involved in this price feed business